core/src/main/scala/org/apache/spark/util/JsonProtocol.scala

```diff
@@ -229,9 +228,11 @@ private[spark] object JsonProtocol {
   def executorMetricsUpdateToJson(metricsUpdate: SparkListenerExecutorMetricsUpdate): JValue = {
     val execId = metricsUpdate.execId
     val taskMetrics = metricsUpdate.taskMetrics
+    val executorMetrics = metricsUpdate.executorMetrics
     ("Event" -> Utils.getFormattedClassName(metricsUpdate)) ~
     ("Executor ID" -> execId) ~
-    ("Metrics Updated" -> taskMetrics.map { case (taskId, stageId, stageAttemptId, metrics) =>
+    ("Executor Metrics Updated" -> executorMetricsToJson(executorMetrics)) ~
+    ("Task Metrics Updated" -> taskMetrics.map { case (taskId, stageId, stageAttemptId, metrics) =>
```

Comment:
unfortunately we can't change the name b/c of the compatibility guarantees we want: (https://github.com/apache/spark/blob/ba882db6f43dd2bc05675133158e4664ed07030a/core/src/main/scala/org/apache/spark/util/JsonProtocol.scala#L39).  I think we're stuck w/ "Metrics Updated" for tasks, and "Executor Metrics Updated" for the other metrics.